modules/social_features/social_user_export/src/Plugin/UserExportPlugin/UserStatus.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\social_user_export\Plugin\UserExportPlugin;
+
+use Drupal\social_user_export\Plugin\UserExportPluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\user\UserInterface;
+
+/**
+ * Provides a 'UserStatus' user export row.
+ *
+ * @UserExportPlugin(
+ *  id = "user_status",
+ *  label = @Translation("Status"),
+ *  weight = -400,
+ * )
+ */
+class UserStatus extends UserExportPluginBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * Returns the header.
+   *
+   * @return \Drupal\Core\StringTranslation\TranslatableMarkup|string
+   *   The header.
+   */
+  public function getHeader() {
+    return $this->t('Status');
+  }
+
+  /**
+   * Returns the value.
+   *
+   * @param \Drupal\user\UserInterface $entity
+   *   The User entity to get the value from.
+   *
+   * @return string
+   *   The value.
+   */
+  public function getValue(UserInterface $entity) {
+    $status = $entity->get('status')->getValue();
+    return !empty($status[0]['value']) ? $this->t('Active') : $this->t('Blocked');
```

Comment:
I suggest we use `return $entity->isActive() ? $this->t('Active') : $this->t('Blocked');`